packages/expo-blur/android/src/main/java/expo/modules/blur/ExpoBlurView.kt

```diff
@@ -88,4 +86,75 @@ class ExpoBlurView(context: Context, appContext: AppContext) : ExpoView(context,
     }
     blurView.invalidate()
   }
+
+  override fun onAttachedToWindow() {
+    super.onAttachedToWindow()
+
+    // Now we can safely walk the parent hierarchy
+    if (!isBlurViewConfigured) {
+      configureBlurView()
+      isBlurViewConfigured = true
```

Comment:
Most likely it should be in reverse order. Otherwise `setBlurRadius, setBlurMethod and applyTint` will early-return and would not set it's values on `blurView`.
```suggestion
      isBlurViewConfigured = true
      configureBlurView()
```